napalm_ce/ce.py

```diff
@@ -768,7 +768,7 @@ def get_mac_address_table(self):
         mac_address_table = []
         command = 'display mac-address'
         output = self.device.send_command(command)
-        re_mac = r"(?P<mac>\S+)\s+(?P<vlan>\d+|-)\S+\s+(?P<interface>\S+)\s+(?P<type>\w+)\s+(?P<age>\d+|-)"
+        re_mac = r"(?P<mac>\S+)\s+(?P<vlan>\d+|-)\s+(?P<interface>\S+)\s+(?P<type>\w+)\s+(?P<age>\d+|-)"
```

Comment:
Hi @nbulat This is get_mac_address_table instead of get_arp_table. Do you have any example of ce outout?